released/packages/coq-mathcomp-classical/coq-mathcomp-classical.1.10.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "A library for classical logic for mathematical components"
+description: """
+This repository contains a library for classical logic for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-C" "classical" "-j%{jobs}%"]
+install: [make "-C" "classical" "install"]
+depends: [
+  ("coq" {>= "8.19" & < "8.21~"}
+  | "rocq-core" { (>= "9.0" & < "9.1~") | (= "dev") })
```

Comment:
Note that your `Makefile` is explicitly invoking `coq_makefile`, so depending on `rocq_core` is incorrect, as it does not provide it. (Fortunately, `coq-hierarchy-builder` does depend on `coq-core`, which hides the issue.)

Reply:
Also, `rocq-hierarchy-builder` depends on `coq-elpi` instead of `rocq-elpi`, so even when `rocq-hierarchy-builder` is used, we get `coq` shim installed (cc @proux01).

Reply:
Indeed, my bad, c.f., https://github.com/rocq-prover/opam/pull/3399
We should probably have a CI job here that warns when any coq-ù package depends on some rocq-* (which is very likely to be an error coming from all the copy&paste required by opam files)